active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,392 @@
+                NextButtonVisibility="HiddenOnEdge"
+                LastButtonVisibility="None"/>
+        </muxc:PagerControl>
+</Grid>
+```
+
+### Code Example
+
+Below is a code sample for how to use the pager control with a data grid and how to handle the page changed event. 
+Since this control is only a UI component, the data managment still needs to be handled through code. 
+This example also shows how to set the number of pages property and manage what images 
+need to be shown to the user when they select a specific page number. 
+
+When a user interacts with the pager control and selects an edge button or page number, the page changed event will fire. 
+This event will provide the index the user is coming from (PreviousPageIndex) and the index the user is navigating to (NewPageIndex). 
+
+XAML
+```XAML
+<Page
+    x:Class="WindowsGallaryApp.Samples.DataGridSamplePage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    Loaded="OnPageLoaded">
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="auto"/>
+        </Grid.RowDefinitions>
+        <controls:DataGrid x:Name="dataGrid" AutoGenerateColumns="True"/>
+        <muxc:ProgressRing x:Name="progressRing" Visibility="Collapsed" IsActive="false">
+        <muxc:Pager x:Name="pager" Grid.Row="1" PageChanged="OnPageChanged" PagerDisplayMode="NumberPanel"/>
+    </Grid>
+</Page>
+```
+C#
+```CSharp
+public sealed partial class DataGridSamplePage : Page
+{
+        private const int imagesPerPage = 25;
+
+        public DataGridSamplePage()
+        {
+                this.InitializeComponent();
+        }
+
+        private async void OnPageLoaded(object sender, RoutedEventArgs e)
+        {
+                pager.NumberOfPages = await ImageGenerator.GetNumberOfPages(imagesPerPage);
+                dataGrid.ItemsSource = await ImageGenerator.GetImagesInPage(0, imagesPerPage);
+                progressRing.Visibility = "Collapsed";
+                progressRing.IsActive = false;
+        }
+        private async void OnPageChanged(Pager sender, PageChangedEventArgs args)
+        {
+                dataGrid.ItemsSource = await ImageGenerator.GetImagesInPage(args.CurrentPage, imagesPerPage);
+                progressRing.Visibility = "Visible";
+                progressRing.IsActive = true; 
+        }
+}
+```
+
+# API Notes
+| Name | Description| Default | 
+|:---:|:---|:---|
+| PagerControlDisplayMode | Enum that contains 4 values (Auto, ComboBox, NumberBox, ButtonPanel) to represent the look of the pager control. When Auto is selected, the display mode will be ComboBox if the NumberOfPages property is less than 11 otherwise it will be NumberBox. | Auto |
```

Comment:
ButtonPanel --> NumericalButtonPanel to match above.

Would love to add BreadbrumbButtonPanel here in the future for the "row of dots" style.